src/main/menu.js

```diff
@@ -563,6 +563,30 @@ gpii.app.menu.getSimpleMenuItem = function (label, event, payload) {
     };
 };
 
+/**
+  * Generates an object that represents a selectable menu item. Similar to getSimpleMenuItem,
+  * but respects an initial boolean which could be a setting from the siteConfig determining
+  * whether to display this item or not.
+  * @param {Boolean} hideItem - If true, this menu item will not be displayed. (ie. this function
+  * will not return the usual menu structure).
+  * @param {String} label - The label of the item.
+  * @param {String} event - The event to be triggered on click.
+  * @param {Object} [payload] - The payload that is to be supplied with the on click event.
+  * @return {ElectronMenuItem} A simple selectable Electron menu item.
+  */
+gpii.app.menu.getOptionalMenuItem = function (hideItem, label, event, payload) {
+    if (hideItem) {
```

Comment:
I'm not a religious fanatic, but single (and explicit) return-ism is slightly preferable - how about 
````
return hideItem ? undefined : {
            label: label,
            click: event,
            args: payload || {}
        };
````